### PR TITLE
Create Informasi-Jadwal-Praktikum.md

### PR DESCRIPTION
**Pull request by MuhammadAlifMaulanaR**

### CODE REVIEW

File: Informasi-Jadwal-Praktikum.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> MUHAMMAD ALIF MAULANA.R - 13020220223
+
+
+> [!IMPORTANT]
+> Key information users need to know to achieve their goal.
+
+## THE PATH
+...
+> PREPARE
+>> Install Discord
```
Comment (MuhammadAlifMaulanaR) [inline Informasi-Jadwal-Praktikum.md:14]: Done to install discord

File: Informasi-Jadwal-Praktikum.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+>> Add Plugin PlantUML
```
Comment (MuhammadAlifMaulanaR) [inline Informasi-Jadwal-Praktikum.md:15]: Done to add plugin plantUML in VSC

Comment (MuhammadAlifMaulanaR) [inline Informasi-Jadwal-Praktikum.md:15]: Done to add plugin plantUML in VSC

Comment (BE2047401) [reply Informasi-Jadwal-Praktikum.md:15]: fix it doesn't still not match the template

**Comment by BE2047401** [comment]
fix it doesn't still not match the template

**Pull request closed without merging**